modules/ROOT/pages/design-common-errors-10.adoc

```diff
@@ -0,0 +1,325 @@
+
+----
+#%RAML 1.0
+title: ExampleRAML
+...
+/rooms:
+  displayName: rooms
+  get:
+    description: get all rooms
+    responses:
+      200:
+        body:
+          application/json:
+            example:
+             {
+               [{
+                "Name": "Superior King",
+                "Number": "201",
+                "Property": "SE030",
+                "Status": "Clean"
+                },
+                {
+                "Name": "Junior Suite",
+                "Number": "202",
+                "Property": "NO131",
+                "Status": "Clean"
+                }]
+              }
+----
+
+If the example was meant be an object, then a key must be specified for it.
+
+----
+#%RAML 1.0
+title: ExampleRAML
+...
+/rooms:
+  displayName: rooms
+  get:
+    description: get all rooms
+    responses:
+      200:
+        body:
+          application/json:
+            example:
+            {
+    	      "some_key": [
+                {
+                  "Name": "Superior King",
+          	  "Number": "201",
+          	  "Property": "SE030",
+          	  "Status": "Clean"
+          	},
+          	{
+          	  "Name": "Junior Suite",
+          	  "Number": "202",
+          	  "Property": "NO131",
+          	  "Status": "Clean"
+          	}
+              ]
+            }
+----
+
+If the example was meant be an array, then the outside curly braces must be removed.
+
+----
+#%RAML 1.0
+title: ExampleRAML
+...
+/rooms:
+  displayName: rooms
+  get:
+    description: get all rooms
+    responses:
+      200:
+        body:
+          application/json:
+            example:
+            [
+                {
+                  "Name": "Superior King",
+          	  "Number": "201",
+          	  "Property": "SE030",
+          	  "Status": "Clean"
+          	},
+          	{
+          	  "Name": "Junior Suite",
+          	  "Number": "202",
+          	  "Property": "NO131",
+          	  "Status": "Clean"
+          	}
+            ]
+----
+
+
+== Common Error 3
+// APIMF-907
+
+*_Not naming named examples_*
+
+The first line in a NamedExample fragment must be a key that is the name given to the example, as in this snippet:
+
+----
+#%RAML 1.0 NamedExample
+MyExampleName:
+----
+
+The properties of the example then follow after. For example, suppose an API specification defines the following object:
+
+----
+user_name:
+    type: object
+    properties:
+        user: string
+----
+
+The NamedExample fragment would need to look like this:
+
+----
+#%RAML 1.0 NamedExample
+MyExampleName:
+    user: Lionel
+----
+
+However, if MyExampleName is missing, then `user` is parsed as the name and `Lionel` is parsed as the example. Because `Lionel` is not surrounded by quotation marks, the parser considers it to be a scalar, which does not match the type of the object that linked to the example.
+
```

Comment:
All plain text are consider scalars in yaml (does not matter if they are quoted or not)

Reply:
From your explanation of APIMF-907:
_In this case, the type of the example value will be scalar, on will not match against the object parsed type._
Did I misunderstand you here?

Reply:
That is right, but no because is quoted or not. That text always gonna be a scalar.

Reply:
Fixed.